src/@canonical/macaroon-bakery.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@canonical/macaroon-bakery";
```

Comment:
Sorry I missed your PR on the bakery lib to move to Typescript!

Reply:
No worries, thanks for the review!